README.md

```diff
@@ -1 +1,145 @@
-# HelloMicroverseProject
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="https://images-platform.99static.com/VhAY_rEp_fvVyR3RNDp3hdWqjww=/1127x2240:1902x3015/500x500/top/smart/99designs-contests-attachments/111/111597/attachment_111597410" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
```

Comment:
- [Optional] I notice you have updated the microverse logo with something similar to what microverse has, This might not be a problem for now considering you are on a "hello microverse" project. But in a future project, please make sure to use the right logo for the project.

Reply:
Hi again @Tadesse-Alemayehu, just to let you know that I followed your suggestions and I put the correct logo of Microverse. Thanks a lot :+1:.